packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -130,11 +130,13 @@ export function normalizeError(
         fluidError.addTelemetryProperties({ typeofError: typeof (error) });
     }
 
-    const originalErrorTelemetryProps = isILoggingError(error) ? error.getTelemetryProperties() : {};
+    const originalErrorTelemetryProps = LoggingError.typeCheck(error)
```

Comment:
nit: rename this variable

Reply:
Sounds good. I have added the changes in commit #a1d49538.